pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -665,9 +669,53 @@ def do_ls_backups(self):
         with self.lock:
             return self.data.get('backups', {})
 
-    def do_update_binaries(self):
+    # Updates the URL for the given epoch height or binary idx. adds a new one if the epoch height does not exit
+    def update_binaries_url(self, neard_binary_url, epoch_height, binary_idx):
+        if neard_binary_url is not None and ((epoch_height is None)
+                                             != (binary_idx is None)):
+            logging.info(
+                f'Updating binary list for height:{epoch_height} or idx:{binary_idx} with '
+                f'url: {neard_binary_url}')
+        else:
+            logging.error(f'wrong arguments provided')
```

Comment:
would be helpful to print the arguments. Also I think here we should be returning an error to the caller of the RPC method. See the instances of `raise jsonrpc.exceptions.JSONRPCDispatchException()` elsewhere in this file

Reply:
Is there a documented list of accepted error codes that I should use?